topics/fair/metadata.yaml

```diff
@@ -5,6 +5,10 @@ title: "FAIR Data, Workflows, and Research"
 summary: "These lessons will teach you how to make your research objects more FAIR with practical, hands-on advice."
```

Comment:
would you like to include 'Management' somewhere in the title?

Reply:
I think it is ok without it we can keep it as a subsection FAIR Data Management